tests/foreman/ui_airgun/test_subscription.py

```diff
@@ -0,0 +1,313 @@
+        with open(temporary_local_manifest_path, 'wb') as file_handler:
+            file_handler.write(manifest.content.read())
+    with session:
+        session.organization.select(org.name)
+        session.subscription.add_manifest(temporary_local_manifest_path)
+        assert session.subscription.has_manifest
+        delete_message = session.subscription.read_delete_manifest_message()
+        assert ' '.join(expected_message_lines) == delete_message
+        assert session.subscription.has_manifest
+        session.subscription.delete_manifest()
+        assert not session.subscription.has_manifest
+
+
+@tier2
+def test_positive_access_with_non_admin_user_without_manifest(test_name):
+    """Access subscription page with user that has only view_subscriptions
+    permission and organization that has no manifest uploaded.
+
+    :id: dab9dc15-39a8-4105-b7ff-ecef909dc6e6
+
+    :expectedresults: Subscription page is rendered properly without errors
+
+    :BZ: 1417082
+
+    :CaseLevel: Integration
+
+    :CaseImportance: Critical
+    """
+    org = entities.Organization().create()
+    role = entities.Role(organization=[org]).create()
+    create_role_permissions(
+        role,
+        {'Katello::Subscription': ['view_subscriptions']}
+    )
+    user_password = gen_string('alphanumeric')
+    user = entities.User(
+        admin=False,
+        role=[role],
+        password=user_password,
+        organization=[org],
+        default_organization=org,
+    ).create()
+    with Session(test_name, user=user.login, password=user_password) as session:
+        assert not session.subscription.search('')
+        assert not session.subscription.has_manifest
+        assert not session.browser.url.endswith('katello/403')
+
+
+@skip_if_bug_open('bugzilla', 1651981)
+@tier2
+@upgrade
+def test_positive_access_with_non_admin_user_with_manifest(test_name):
+    """Access subscription page with user that has only view_subscriptions
+    permission and organization that has a manifest uploaded.
+
+    :id: 9184fcf6-36be-42c8-984c-3c5d7834b3b4
+
+    :expectedresults: Subscription page is rendered properly without errors
+        and the default subscription is visible
+
+    :BZ: 1417082, 1651981
+
+    :customerscenario: true
+
+    :CaseLevel: Integration
+
+    :CaseImportance: Critical
+    """
+    org = entities.Organization().create()
+    manifests.upload_manifest_locked(org.id)
+    role = entities.Role(organization=[org]).create()
+    create_role_permissions(
+        role,
+        {'Katello::Subscription': ['view_subscriptions']}
+    )
+    user_password = gen_string('alphanumeric')
+    user = entities.User(
+        admin=False,
+        role=[role],
+        password=user_password,
+        organization=[org],
+        default_organization=org,
+    ).create()
+    with Session(test_name, user=user.login, password=user_password) as session:
+        session.subscription.navigate_to(session.subscription, 'All')
```

Comment:
same here
and basically `assert not session.browser.url.endswith('katello/403')` doesn't make sense if you do search afterwards like in previous test case

And I read both 1417082, 1651981 BZ. We do not need to verify `assert not session.browser.url.endswith('katello/403')` explicitly. If search is working for both tests then necessary page showing content.
so for case above do:
        assert not session.subscription.search('')
        assert not session.subscription.has_manifest

and for this:
        assert (session.subscription.search(
            'name = "{0}"'.format(DEFAULT_SUBSCRIPTION_NAME))[0]['Name']
                == DEFAULT_SUBSCRIPTION_NAME)